tests/functional/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -197,3 +208,210 @@ def test_add_capacity_pre_upgrade(self, reduce_and_resume_cluster_load):
         Test to add variable capacity to the OSD cluster while IOs running
         """
         add_capacity_test()
+
+
+@turquoise_squad
+@stretchcluster_required
+class TestAddCapacityStretchCluster:
+    """
```

Comment:
Add all the relevant markers where the test is not expected to run, eg: bm, non lso, external mode, managed service etc

Reply:
**stretchcluster_required** marker makes sure that test wont be run in any other environments except stretch cluster